drivers/usb/device/usb_dc_stm32.c

```diff
@@ -53,6 +54,19 @@
 #define SYS_LOG_LEVEL CONFIG_SYS_LOG_USB_DRIVER_LEVEL
 #include <logging/sys_log.h>
 
+/*
+ * USB LL API provides the EP_TYPE_* defines STM32Cube does
+ * not provide USB LL API for STM32F0 and STM32F3 families.
+ * Map EP_TYPE_* defines to PCD_EP_TYPE_* defines
+ */
+#if defined(CONFIG_SOC_SERIES_STM32F3X) || defined(CONFIG_SOC_SERIES_STM32F0X)
+#define EP_TYPE_CTRL PCD_EP_TYPE_CTRL
+#define EP_TYPE_ISOC PCD_EP_TYPE_ISOC
+#define EP_TYPE_BULK PCD_EP_TYPE_BULK
+#define EP_TYPE_INTR PCD_EP_TYPE_INTR
+#endif
+/* CONFIG_SOC_SERIES_STM32F3X */
```

Comment:
Is there a reason for having this comment alone on this line?

Reply:
Fixed